uwrt_mars_rover_blackfly_calibration/calibration_data/left_18475242.yaml

```diff
@@ -0,0 +1,20 @@
+image_width: 1440
+image_height: 1080
+camera_name: left_18475242
```

Comment:
this is the number written on the particular blackfly i have

Reply:
it actually said left? is the number after the serial or smth? is there a way in spinnaker to change it?

Reply:
i meant that there's a sticker that someone put on the camera that reads "left 18475424"